<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>434</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Eye Lids Rig Helper</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The Vertices must be selected in order, start from the most near to the nose.&lt;/p&gt;&lt;p&gt;You can leave the upper or the lower Lids verteces blank, if there are no eyelids there.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Upper Lids Verteces:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="fillUpperLidsVerteces_btn">
        <property name="text">
         <string>&lt;&lt;&lt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="selectUpperLids_btn">
        <property name="text">
         <string>Select</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QListWidget" name="upperLidsVerteces_lw"/>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>* Lower Lids some times start from same vertex than upper lids</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Lower Lids Verteces:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="fillLowerLidsVerteces_btn">
        <property name="text">
         <string>&lt;&lt;&lt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="selectLowerLids_btn">
        <property name="text">
         <string>Select</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QListWidget" name="lowerLidsVerteces_lw"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Center Pivot Object:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="centerPivot_le"/>
      </item>
      <item>
       <widget class="QPushButton" name="fillCenterPivot_btn">
        <property name="text">
         <string>&lt;&lt;&lt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QCheckBox" name="mirror_chb">
        <property name="text">
         <string>Mirror</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="create_btn">
        <property name="text">
         <string>Create Eyelids Rig</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
   <zorder>upperLidsVerteces_lw</zorder>
   <zorder>lowerLidsVerteces_lw</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
